components/Layout.vue

```diff
@@ -101,14 +100,14 @@ import { ref, onMounted, onUnmounted } from 'vue';
 const { path } = useRoute();
 const contentPath = path.endsWith('/') ? path.slice(0, -1) : path;
 const contentDirectory = await queryContent().where({ _path: contentPath }).findOne();
-const imageSrc = contentDirectory.img;
+const imageSrc = contentDirectory.img || '/background.png';
```

Comment:
@Wulingoh Minor nit on the name of this file - it's not very descriptive. Could be `default-cover-image.png` or something instead

Reply:
Updated the image name to be more descriptive